louis/db/__init__.py

```diff
@@ -26,7 +28,7 @@ def raise_error(message):
 
 def connect_db():
     """Connect to the postgresql database and return the connection."""
-    # print(f"Connecting to {LOUIS_SCHEMA}")
+    print(f"Connecting to {LOUIS_DSN}")
```

Comment:
use the logging facility instead

Reply:
Could you give me a bit more details ? I don't really understand what you're referring to by "logging facility"

Reply:
there's two lines a few lines above:

```
LOGGER = logging.getLogger(__name__)
logging.basicConfig(level=logging.DEBUG)
```

this is a logging facility

some doc:
https://gist.github.com/andreasWallner/c7e677eaade39b49a70b

Reply:
(I would make an exception and lowercase LOGGER to logger as it will be sprinkled across your code)

Reply:
done